EasySockets/EasySockets/Builder/ReadonlyEasySocketOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+
+namespace EasySockets.Builder;
+
+public struct ReadonlyEasySocketOptions
+{
+    public ReadonlyEasySocketOptions(EasySocketOptions options)
+    {
+        ReceiveBufferSize = options.ReceiveBufferSize;
+        SendBufferSize = options.SendBufferSize;
+        Encoding = options.Encoding;
+        ClosingStatusDescription = options.ClosingStatusDescription;
+    }
```

Comment:
Creates a dependency on the EasySocketOptions, could be removed since an extension method is used anyway to map the corresponding values